src/app/upload_services/BaseUploadService.py

```diff
@@ -5,16 +5,18 @@
 
 from werkzeug.datastructures import FileStorage
 
+from . import upload_logger
+
 
 class BaseUploadService(ABC):
     def __init__(self, datatype: str):
-        self.EXT_REGEX = os.environ.get("EXT_REGEX", "json")
+        self.EXT_REGEX = os.environ.get("EXT_REGEX", ".json")

```

Comment:
Dollar sign at the end says to ONLY match if the string ends with that. 

```suggestion
        self.EXT_REGEX = os.environ.get("EXT_REGEX", ".json$")

```

Reply:
removed the . and added a $